docs/app/pages/showcase/visualizations/partition-map/partition-map.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="chart-container">
+    <ux-partition-map class="partition-map" [dataset]="dataset" [colors]="colors">
+
+        <ng-template
```

Comment:
Could we add some styling for the popover? Currently it looks like this:

![image](https://user-images.githubusercontent.com/20795331/72165086-c29b9380-33be-11ea-8b7b-220730b65378.png)

Whereas in MF docs it is like this:

![image](https://user-images.githubusercontent.com/20795331/72165106-cb8c6500-33be-11ea-8457-246248992233.png)